app/src/main/java/com/example/georgesamuel/dubaihotels/adapter/HotelAdapter.java

```diff
@@ -0,0 +1,83 @@
+package com.example.georgesamuel.dubaihotels.adapter;
+
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.example.georgesamuel.dubaihotels.activities.DetailActivity;
+import com.example.georgesamuel.dubaihotels.R;
+import com.example.georgesamuel.dubaihotels.model.Hotel;
+import com.google.android.material.card.MaterialCardView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HotelAdapter extends RecyclerView.Adapter<HotelAdapter.HotelViewHolder>{
+
+    private Context context;
+    private List<Hotel> hotelsList = new ArrayList<>();
+
+    public HotelAdapter(Context context, List<Hotel> list){
+        this.context = context;
+        this.hotelsList = list;
+    }
+
+    @NonNull
+    @Override
+    public HotelViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.hotel_item, parent, false);
+        return new HotelViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull HotelViewHolder holder, int position) {
+
+        Glide.with(context).load(hotelsList.get(position).getImage().get(0).getUrl()).into(holder.hotelImage);
+        holder.hotelName.setText(hotelsList.get(position).getSummary().getHotelName());
+
+
+        holder.cardView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+
+                Intent intent = new Intent(context, DetailActivity.class);
+                intent.putExtra("url", hotelsList.get(position).getImage().get(0).getUrl());
```

Comment:
put static words like url , name , address ..etc as final static strings to be reusable between activities , please see code style section 2.2.13 String constants, naming, and values
https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md